### ISSUE TITLE
2307 supported ? 

### ISSUE THREAD

**Issue by Javier-bat** [brandonlw/Psychson#211]
is 2307 controller actually supported?  

![image](https://user-images.githubusercontent.com/32001377/58373981-85a48e80-7f0c-11e9-82ed-011d86e2b675.png)

**Comment by p00n00b** [comment]
Yes https://github.com/p00n00b/2307-badusb/blob/master/README.md

**Comment by Ironozd** [comment]
@p00n00b Can you confirm right now that it is working?

**Comment by Ironozd** [comment]
If yes, will buy today
